Algorithmia/algo_response.py

```diff
@@ -18,21 +18,21 @@ def __repr__(self):
             return self.__unicode__().encode('utf-8')
 
     @staticmethod
-    def create_algo_response(responseJson):
```

Comment:
Honestly if anyone has a better idea on how to quickly escape in the event that we recieve an unformatted 5xx HTTP response instead of a json parsable one coming from the Algorithmia API server; I'll implement it! From what I can see this is the least intrusive change required